docs/docs/guides/template-backup-script.md

```diff
@@ -39,6 +23,16 @@ BACKUP_PATH="/path/to/local/backup/directory"
 REMOTE_HOST="remote_host@IP"
 REMOTE_BACKUP_PATH="/path/to/remote/backup/directory"
 
+### init Borg setup
+if [ ! -d "$UPLOAD_LOCATION/database-backup" ]; then
+    mkdir "$UPLOAD_LOCATION/database-backup"
```

Comment:
We need to check for success of mkdir here since the rest of the script would now depend on it